bin/build

```diff
@@ -52,45 +46,31 @@ if [ -n "$REBUILD_RPM" ]; then
 fi
 
 pushd $IMAGE_DIR
-  cmd="$CONTAINER_COMMAND build"
-
-  # --format docker is needed for podman to ensure the builds are built with docker v2 manifests
-  if [ "$CONTAINER_COMMAND" == "podman" ]; then
-    cmd+=" --format docker"
-  fi
-
-  cmd+=" --tag $REPO/manageiq-base:$TAG \
-         --pull \
-         --build-arg RPM_PREFIX=$RPM_PREFIX"
-
-  if [ -n "$NO_CACHE" ]; then
-    cmd+=" --no-cache"
-  fi
-
-  if [ -n "$LOCAL_RPM" ] || [ -n "$REBUILD_RPM" ]; then
-    cmd+=" --build-arg LOCAL_RPM=true"
-  fi
+  # Pull the base image first
+  $CONTAINER_COMMAND pull registry.access.redhat.com/ubi9/ubi
 
-  if [ -n "$RELEASE_BUILD" ]; then
-    cmd+=" --build-arg RELEASE_BUILD=true"
-  fi
-
-  echo "Building manageiq-base: $cmd"
-  $cmd manageiq-base
-
-  build_images="manageiq-base-worker manageiq-orchestrator manageiq-webserver-worker manageiq-ui-worker"
+  build_images="manageiq-rpms manageiq-base manageiq-base-worker manageiq-orchestrator manageiq-webserver-worker manageiq-ui-worker"
   for image in $build_images; do
-    cmd="$CONTAINER_COMMAND build"
+    cmd="$CONTAINER_COMMAND build --build-arg FROM_REPO=$REPO --build-arg FROM_TAG=$TAG --build-arg RPM_PREFIX=$RPM_PREFIX --tag $REPO/$image:$TAG"
+
+    if [ -n "$VERBOSE" ]; then
+      cmd+=" --progress plain"
+    fi
```

Comment:
This is mostly just for debugging with the new docker output format via docker buildx.  I can remove this if you want.